### ISSUE TITLE
Show fields prior to choosing GitHub repo

### ISSUE THREAD

**Issue by LorenHeyns** [codeforamerica/publiccode-pusher#13]
To make this a great demo, we need to continue showing the form prior to selecting a GitHub repo. Doing so will encourage broader adoption.

Here's the visible form in the initial Italian work:
https://publiccode-editor.developers.italia.it

Here's our new, hard-to-preview form:
https://codeforamerica.github.io/publiccode-pusher

In our new version, we could display a box at the top:
**Select a Github Repo to update. "Select a Repo" "Learn More"**

Initially, we could prevent editing prior to choosing a repo.

Later, we could store the values as local data prior to choosing a repo - so the fields remain populated with backing up or when opening the page later in another tab.

**Comment by giosce** [comment]
I agree. I didn't see that Italian version, I saw another one and also I forked from the Italian repo.
What @LorenHeyns suggest makes sense.
There is a big difference though. Our version has a "Push to Github" button that creates a branch and PR on the logged in repo. This is very useful, the Italian version instead it seems it will generate a local yaml file (sure, not difficult to upload it manually to github or anywhere else).
Maybe multiple buttons...? ("Generate", "Login", "Push")?
